client/utils/challenge-types.js

```diff
@@ -11,7 +11,8 @@ const quiz = 8;
 const invalid = 9;
 const pythonProject = 10;
 const video = 11;
-const codeally = 12;
+const codeAllyPractice = 12;
+const codeAllyCert = 13;
```

Comment:
It seems like we have both `codeAlly` challenge types, as well as `codeRoad` challenge types.

Edit: Never mind, it is just the comments that mention CodeRoad.